src/features/MenuPage/index.tsx

```diff
@@ -0,0 +1,46 @@
+import { Link } from 'react-router-dom';
+import { PLANET_OBSERVER_PATH } from 'src/features/Router/path';
+import { CONTACT_US_PATH } from 'src/features/Router/path';
+import { ABOUT_US_PATH } from 'src/features/Router/path';
+import { SOLAR_SYSTEM_PATH } from 'src/features/Router/path';
+import { PROPOSE_CHANGES_PATH } from 'src/features/Router/path';
+import { SAY_THANKS_PATH } from 'src/features/Router/path';
```

Comment:
where css for this page?

Reply:
it's better to create "menu component" and use css there.

Reply:
Done